browser/ui/webui/settings/brave_appearance_handler.cc

```diff
@@ -74,7 +78,10 @@ void BraveAppearanceHandler::RegisterMessages() {
       kNewTabPageSuperReferralThemesOption,
       base::BindRepeating(&BraveAppearanceHandler::OnPreferenceChanged,
       base::Unretained(this)));
-
+  profile_state_change_registrar_.Add(
```

Comment:
Maybe we want to use new handler such as `BraveSettingsNTPHandler`? :)

Reply:
That would be really nice - given the scope of this PR, maybe I can capture this and the other fix up item that @petemill mentioned as a follow up

Reply:
Yeah this, and the NTP widget items don't belong in something called `BraveAppearanceHandler`. We can rename it, move these to a new handler, or just add the instant sites handler to the settings page.